cli/tests/test_diff_command.rs

```diff
@@ -787,65 +788,68 @@ fn test_diff_stat() {
 #[test]
 fn test_diff_stat_long_name_or_stat() {
     let mut test_env = TestEnvironment::default();
-    test_env.add_env_var("COLUMNS", "50");
+    test_env.add_env_var("COLUMNS", "30");
     test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
     let repo_path = test_env.env_root().join("repo");
 
     let get_stat = |test_env: &TestEnvironment, path_length: usize, stat_size: usize| {
         test_env.jj_cmd_success(&repo_path, &["new", "root"]);
-        let mut name = "abcdefghij".repeat(path_length / 10 + 1);
-        name.truncate(path_length);
+        let name = "一二三四五六七八九十"
```

Comment:
nit: CJK chars have different problem since 1 char may occupy 2 terminal width. We'll need to use `unicode-width`. Maybe I can fix that in a follow-up PR.

BTW, it's probably better to include file names of various lengths to test the alignment.

Reply:
True about the terminal width for CJK characters (I guess the ones I used here use 1 terminal character, though?).

Reply:
They're Han characters, and are 2-width.

Anyway, new version looks good to me, thanks.